WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -295,8 +306,10 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     } else if (indexPath.section == TableViewSectionPublishType) {
         switch (indexPath.row) {
             case BlogDetailsRowBlogPosts:
-                [WPAnalytics track:WPAnalyticsStatOpenedPosts];
-                controllerClass = [PostsViewController class];
+            {
+                [self showPostList];
+                return;
+            }
                 break;
             case BlogDetailsRowPages:
                 [WPAnalytics track:WPAnalyticsStatOpenedPages];
```

Comment:
I know this wasn't changed here, but maybe (for consistency's sake), braces for every case?.

(Sorry!, annoying ^)